Shared/ViewModels/PagingLibraryViewModel.swift

```diff
@@ -28,6 +29,32 @@ class PagingLibraryViewModel: ViewModel {
         return UIScreen.main.maxChildren(width: libraryGridPosterType.width, height: height)
     }
 
+    public func getRandomItemFromLibrary(completion: @escaping (Response<BaseItemDtoQueryResult>) -> Void) {
```

Comment:
Since this is a _new_ async function, let's move away from completion handlers. Make this function `async` and have the button wrap it in its own `Task` and just return the random item.

Yes, many functions are purely async and have a `Task`-wrapped body (that could be migrated) but that was done to just make the `Publisher` -> `async/await` migration possible.